docs/guests.md

```diff
@@ -1,4 +1,50 @@
-# Guest tools
+---
+sidebarDepth: 2
+---
+## VM
+### Windows VM
+#### Manage screen resolution
+##### Bios VM
+For Bios VM screen resolution can be managed directly through the OS:
+- Right click on the desktop
+- Display settings
+- Choose resolution
+
+##### UEFI VM
+
+For UEFI VM you will need to set up your resolution in the UEFI settings of the VM.
+For that you first will need to enter the VM UEFI setup:
+- At VM start during Tiano firware logo display press ESC.
+  
+![](../assets/img/screenshots/VM_Tiano.png)
+- You will enter The UEFI firmware management interface. Navigating with keyboard arrow select Device Manager and press enter.
+  
+![](../assets/img/screenshots/VM_Tiano_1.png)
```

Comment:
Could you replace this screenshot with one that actually shows the "Device Manager" text highlighted? The current one has the language selection highlighted instead.

Reply:
updated